plugins/outputs/groundwork/groundwork_test.go

```diff
@@ -51,7 +53,7 @@ func TestWriteWithDefaults(t *testing.T) {
 	}))
 
 	i := Groundwork{
-		Log:            testutil.Logger{},
+		Log:            new(testutil.Logger),
```

Comment:
Please just keep the old way to minimize the diff. Same for the instances below.

Reply:
done